### PR TITLE
Update node-tagging.md

### PR DESCRIPTION
**Pull request by zyoung51**
@heckj @amymullins

### CODE REVIEW

**Comment by heckj** [comment]
@zyoung51 looks great, exactly what we talked about, and I think will match a lot of the requests we're seeing from folks like rlarkin :+1:

File: node-tagging.md [hunk 1]
```diff
@@ -2,28 +2,25 @@
...
-GET /tags/:name - Get information about a tag
-GET /tags/:name/nodes - Get a list of nodes that have the tag
-PATCH /tags/:name - update a tag's add/remove workflows/settings
-DELETE /tags/:name - Delete a tag
+* PATCH /nodes/:nodeid/tags - Add a list of tags to a node
+* GET /nodes/:nodeid/tags - Get a list of tags on the node
+* DELETE /nodes/:nodeid/tags/:name - Remove a tag from a node
+* POST /tags - Create a tag with a set of rules
+* GET /tags - Get a list of all tags
+* GET /tags/:name - Get information about a tag
```
Comment (yyscamper) [inline node-tagging.md]: Could you please give some example what kind of information a tag will have except the `name`, `createdDate`?

**Comment by yyscamper** [comment]
:+1:

File: node-tagging.md (hunk 1 above)
Comment (zyoung51) [inline node-tagging.md]: The 'tag' model is identical to the 'sku' model.  It also contains a set of 'rules' that describe paths to match in the catalogs.

**Comment by amymullins** [comment]
+1

**Pull request merged**